ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -3330,6 +3330,55 @@ def test_paddle_tensor_squeeze_(
     )
 
 
+# stack
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="stack",
+    dtype_value=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric").filter(
```

Comment:
Couldn't we just use the supported dtypes decorator on the function instead of doing the filtering here?

Reply:
Resolved @bipinKrishnan ..Thanks for the review